guide/README.md

```diff
@@ -1895,7 +1869,7 @@ catch up with the conductor. The No-clock block essentially says, do
 your own thing and don't worry about the conductor.
 
 ![alt
- tag](https://rawgithub.com/sugarlabs/musicblocks/f057d42544ff53dc3d71ee241b53bf6bff0ddf24/documentation/no-clock.svg
+ tag](.../f057d42544ff53dc3d71ee241b53bf6bff0ddf24/documentation/no-clock.svg
```

Comment:
It seems as if the wrong image (on a branch?) was being referenced. Is there a "no-clock" image in the master documentation directory? If not, we should add one.

Reply:
No there is no 'no-clock' image in master documentation. I will add one and update the pull request.